Makefile

```diff
@@ -47,3 +48,16 @@ check_env:
 		echo "updates_repo_url is undefined, Please check README"; \
 		exit 1; \
 	fi
+
+.PHONY: get_gh-release
+get_gh-release: init
+	curl -sL https://github.com/progrium/gh-release/releases/download/v2.2.1/gh-release_2.2.1_linux_x86_64.tgz > $(BUILD_DIR)/gh-release_2.2.1_linux_x86_64.tgz
+	tar -xvf $(BUILD_DIR)/gh-release_2.2.1_linux_x86_64.tgz -C $(BUILD_DIR)
+	rm -fr $(BUILD_DIR)/gh-release_2.2.1_linux_x86_64.tgz
+
+.PHONY: release
+release: centos_iso get_gh-release
+	mkdir -p release
+	cp $(BUILD_DIR)/minishift-centos.iso release/
+	$(BUILD_DIR)/gh-release checksums sha256
+	$(BUILD_DIR)/gh-release create minishift/minishift $(VERSION) master v$(VERSION)
```

Comment:
_minishift/minishift_, that's not quite right. It's _minishift/minishift-centos-iso_, right.